### DISCUSSION TITLE
救救孩子! 麻烦站长大大看看, 我这个真不是NSFW, 是学术内容😭😭😭😭

### DISCUSSION THREAD

**Discussion by nmdwgll** [BurnHair/docs#115] [技术支持]
![微信截图_20240621182228](https://github.com/BurnHair/docs/assets/11284864/8e240eea-7043-425e-8ade-40930e209f44)

**Comment by BennyThink** [comment]
🤣等我有空改改

**Comment by BennyThink** [comment]
再试试看，我把一些术语去掉了

**Reply by nmdwgll** [reply]
应该可以了, 感恩~~🙏
